src/Infrastructure/Security/SymfonyUser.php

```diff
@@ -55,9 +55,17 @@ public function getPassword(): string
         return $this->password;
     }
 
-    public function getOrganization(): Organization
+    public function getOrganizations(): array
     {
-        return $this->organization;
+        return $this->organizations;
+    }
+
+    public function getOrganizationUuids(): array
+    {
+        return array_map(
+            function (Organization $organization) { return $organization->getUuid(); },
```

Comment:
J'ai appris que PHP avait cette syntaxe de lambda "simplifiée"


```suggestion
            fn (Organization $organization) => $organization->getUuid(),
```